Add fetch-events support to Exabeam Security Operations Platform on XSIAM (#40990)
demisto/content: Packs/ExabeamSecurityOperationsPlatform/Integrations/ExabeamSecOpsPlatform/README.md

## Configure Exabeam Security Operations Platform in Cortex

| **Parameter** | **Description** | **Required** |
| --- | --- | --- |
| Server URL | | True |
| Client ID | | True |
| Client Secret | | True |
| Trust any certificate (not secure) | | False |
| Use system proxy settings | | False |
| Fetch incidents | Supported on Cortex XSOAR only. | False |
| First fetch timestamp (&lt;number&gt; &lt;time unit&gt;, e.g., 12 hours, 7 days) | Supported on Cortex XSOAR only. | False |

> | Maximum Incidents Per Fetch | Supported on Cortex XSOAR only. This value should not exceed 3,000. | False |

Reviewer: ```suggestion
| Maximum Incidents Per Fetch | Supported on Cortex XSOAR only. This value should not exceed 3,000 due to product's API limitations. | False |
```

Author: Updated

| Fetch query | Supported on Cortex XSOAR only. In the `key:value`. For example: `NOT stage:"CLOSED"`. | False |
| Incident type | Supported on Cortex XSOAR only. | False |

> | Fetch events | Supported on Cortex XSIAM and Platform only. | False |

Reviewer 2: ```suggestion
| Fetch events | Supported on Cortex XSIAM versions 2x and 3x only. | False |
```

> | Maximum Events Per Fetch | Supported on Cortex XSIAM and Platform only. This value should not exceed 30,000. | False |

Reviewer 2: ```suggestion
| Maximum Events Per Fetch | Supported on Cortex XSIAM only. This value should not exceed 30,000. | False |
```

Author: Fixed
